mechanical/cad/cad-getting-started.md

```diff
@@ -0,0 +1,58 @@
+## Getting started with Onshape
+
+### Why Onshape
+Team 1675 has standardized on using Onshape as our CAD package to design robots. While this particular CAD software is not currently an industry standard, there are many advantages that it provides our team that most other CAD options are lacking.
+
+1. Full free access for students
+
+    Onshape provides an educational liscense free of charge for any students and teachers. This allows full access to all features with minimal overhead.
+
+2. Low system requirements
+
+    Onshape is a browser-based CAD solution, which means the heavy lifting of is done in the cloud. This allows us to run CAD on nearly any computer and has the additional benefit of not needing to install software to view CAD models.
+
+3. Automatic Syncing and Version Control
+
+    Because it is a cloud-based solution, updates made on one computer can almost instantly be seen on another, without the need for syncing or downloading files. (Think google-docs for CAD). In additional there are version control features that will allow rolling-back changes when needed.
+
+### Creating an Account
+
+1. Using your preferred web browser, navigate to [Onshape.com](https://onshape.com/)
+
+2. Click "Create a Student Account" 
+
+![Onshape Main](/mechanical/cad/cad-images/Onshape_Main.PNG)

```

Comment:
```suggestion
![Onshape Main](./cad-images/Onshape_Main.PNG)

```

(and all other images)

Without the `./` may also work but this explicitly says "from this directory, go to..."
This file is already in `mechanical/cad`